blob/fileblob/fileblob.go

```diff
@@ -606,7 +617,104 @@ func (b *bucket) Delete(ctx context.Context, key string) error {
 	return nil
 }
 
+// SignedURL takes a context, an object key, and a SignedURLOptions struct
```

Comment:
Nit: This is a function on a private type (bucket), so it won't make it to godoc. You can just say `SignedURL implements driver.SignedURL` or something succinct like that.